mindsdb/integrations/handlers/replicate_handler/replicate_handler.py

```diff
@@ -0,0 +1,143 @@
+import replicate
+import pandas as pd
+from mindsdb.integrations.libs.base import BaseMLEngine
+from typing import Dict, Optional
+import os
+import types
+from mindsdb.utilities.config import Config
+
+
+class ReplicateHandler(BaseMLEngine):
+    name = "replicate"
+
+    @staticmethod
+    def create_validation(target, args=None, **kwargs):
+        if 'using' not in args:
+            raise Exception("Replicate engine requires a USING clause! Refer to its documentation for more details.")
+        else:
+            args = args['using']
+
+        if 'model_name' and 'version' not in args:
```

Comment:
This conditional will not work as-is. You need to check that `model_name` is not in `args` too:

```python
if 'model_name' not in args or 'version' not in args 
```